GitUI/CommitInfo/CommitInfoHeader.cs

```diff
@@ -124,28 +125,30 @@ private void rtbRevisionHeader_LinkClicked(object sender, LinkClickedEventArgs e
         {
             var link = _linkFactory.ParseLink(e.LinkText);
 
-            if (Uri.TryCreate(link, UriKind.Absolute, out var uri))
+            if (!Uri.TryCreate(link, UriKind.Absolute, out var uri))
             {
-                if (uri.Scheme == "gitext")
+                return;
+            }
+
+            if (uri.Scheme == "gitext")
+            {
+                CommandClicked?.Invoke(sender, new CommandEventArgs(uri.Host, uri.AbsolutePath.TrimStart('/')));
```

Comment:
[nit] If you add `return` here, we can remove the following `else` and remove one level of nesting.